proto/storage/product_usage.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+option go_package = "storage";
+option java_package = "io.stackrox.proto.storage";
+
+import "google/protobuf/timestamp.proto";
+import "gogoproto/gogo.proto";
+
+package storage;
+
+// SecuredUnits represents a record of an aggregated secured clusters usage
+// metrics. The metrics are aggregated periodically, and put into the database.
+message SecuredUnits {
+    // id is not used to retrieve data, but serves mostly for compatibility with
+    // the current implementation of the query generator.
+    string id                           = 1 [(gogoproto.moretags) = 'sql:"pk,type(uuid)"'];
+    // timestamp stores the moment at which the values of the metrics below are
+    // aggregated.
+    google.protobuf.Timestamp timestamp = 2;
```

Comment:
nit: `last_updated` or `update_timestamp` or `update_ts`. It would be good if we can remove the need to read the comments by choosing names that convey the purpose clearly.

Reply:
Timestamp is the timestamp of the data record. At what time it was recorded. So the name should be good.

Reply:
It could, but there's only one.